islandora8/conf/solrconfig_extra.xml

```diff
@@ -217,4 +217,4 @@
   7.0.0
 -->
   <searchComponent name="tvComponent" class="solr.TermVectorComponent"/>
-<lib dir="/var/solr/data/contrib/ocrhighlighting/lib" regex=".*\.jar" />
+<lib dir="/opt/solr/contrib/ocrhighlighting/lib" regex=".*\.jar" />
```

Comment:
Do the property thing that https://github.com/discoverygarden/solr-image/pull/15/files#r2310382429 suggested and https://github.com/discoverygarden/solr-image/pull/15/files#r2310285657 didn't follow, but also set the default:

```suggestion
<lib dir="${solr.hocr.plugin.path:/opt/solr/contrib/ocrhighlighting/lib}" regex=".*\.jar" />
```

Reply:
From testing it's not picking up the java property with just an env var, so added to solr opts as well. The controller overrides `SOLR_OPTS` so using the `solr.in.sh` to add it.

Reply:
Looking into things, looks like the "automapping" of `SOLR_*` environment variables might be a newer thing, with Solr 9:
- 8.11 docs don't mention it: https://solr.apache.org/guide/8_11/configuring-solrconfig-xml.html#substituting-properties-in-solr-config-files
- latest(/9.9) docs do: https://solr.apache.org/guide/solr/latest/configuration-guide/property-substitution.html

So yeah, is fine.